server/flyway/sql/V34__migrate_fam_role_assignment_to_admin_table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE UNIQUE INDEX fam_usr_app_admin_uk ON app_fam.fam_application_admin(user_id, application_id);
```

Comment:
think we need to add the unique index constrains, so user admin record won't have any duplicate

Reply:
thanks Ian, I didn't notice the naming convention, will update